hazelcast/src/test/java/com/hazelcast/datastore/impl/ExternalDataStoreServiceImplTest.java

```diff
@@ -104,30 +113,30 @@ private ResultSet executeQuery(DataSource dataSource, String sql) throws SQLExce
     }
 
     @Test
-    public void should_fail_when_datastore_class_DOES_NOT_implements_ExternalDataStoreFactory() throws Exception {
+    public void should_fail_when_datastore_class_DOES_NOT_implements_ExternalDataStoreFactory() {
         Config wrongConfig = smallInstanceConfig();
 
         ExternalDataStoreConfig externalDataStoreConfig = new ExternalDataStoreConfig()
                 .setName("wrong-class-name")
                 .setClassName("java.lang.Object");
         wrongConfig.addExternalDataStoreConfig(externalDataStoreConfig);
 
-        assertThatThrownBy(() -> createHazelcastInstance(wrongConfig))
+        assertThatThrownBy(() -> hazelcastInstanceFactory.newHazelcastInstance(wrongConfig))
                 .isInstanceOf(HazelcastException.class)
                 .hasMessage("External data store 'wrong-class-name' misconfigured: 'java.lang.Object'"
                         + " must implement 'com.hazelcast.datastore.ExternalDataStoreFactory'");
     }
 
     @Test
-    public void should_fail_when_datastore_class_NON_existing() throws Exception {
+    public void should_fail_when_datastore_class_NON_existing() {
```

Comment:
nit: can you change NON to normal case? not directly from this change, but little boyscout refactor

Reply:
It's my way of making sure that negation is visible ;)